aries-site/src/pages/templates/ascending-navigation.mdx

```diff
@@ -0,0 +1,45 @@
+import { DiagramPreview, Example } from '../../layouts';
+
+When an application’s hierarchy is well designed, the ascending button will typically take the user 
+back to the previous page they came from.
+
+<Example 
+  caption="Example of an acending button." 
+  plain 
+  pad="small"
+  previewWidth="medium"
+  height={{ min: 'small' }}
+>
+  <DiagramPreview 
+    src={{
+      dark: "/templateImages/ascending-button-preview-invert.svg",
+      light: '/templateImages/ascending-button-preview.svg'
+    }} 
+    alt="image of a ascending button" />
+</Example>
+
+## Guidance
+- Follows the default button styling with a "FormPrevious" icon.
```

Comment:
I know this terminology came up in today's share out. I'm curious if there can be confusion with using "FormPrevious" as the icon name as it isn't remotely close to a form. In Figma, this icon is called "Previous". I'm just wondering if its called the same in the theme? Or is it "FormPrevious"?

Reply:
This is where it's confusing which icon to use there is an icon that is "Previous" as well as "FormPrevious"